Apsitvarkom.DataAccess/IGeocoder.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Apsitvarkom.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Apsitvarkom.DataAccess
+{
+    public interface IGeocoder
+    {
+        Task<string> ReverseGeocode(Coordinates cords);
```

Comment:
`ReverseGeocodeAsync` instead of `ReverseGeocode` to indicate this is an asynchronous method